Tasks/JenkinsQueueJob/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 2,
     "Minor": 126,
```

Comment:
should we bump the minor to 127 or 128 (next week)?

Reply:
Bump to 128, not worth porting to 127